packages/openid4vc/src/openid4vc-holder/OpenId4VciHolderService.ts

```diff
@@ -382,27 +316,22 @@ export class OpenId4VciHolderService {
     options: {
       resolvedCredentialOffer: OpenId4VciResolvedCredentialOffer
       acceptCredentialOfferOptions: OpenId4VciAcceptCredentialOfferOptions
-      resolvedAuthorizationRequestWithCode?: OpenId4VciResolvedAuthorizationRequestWithCode
-      accessToken?: string
+      accessToken: string
       cNonce?: string
-      dpop?: { jwk: Jwk; nonce?: string }
+      dpop?: OpenId4VciDpopRequestOptions
       clientId?: string
     }
   ) {
     const { resolvedCredentialOffer, acceptCredentialOfferOptions } = options
-    const { metadata, version, offeredCredentialConfigurations } = resolvedCredentialOffer
-
-    const { credentialsToRequest, credentialBindingResolver, verifyCredentialStatus } = acceptCredentialOfferOptions
+    const { metadata, offeredCredentialConfigurations } = resolvedCredentialOffer
+    const { credentialConfigurationIds, credentialBindingResolver, verifyCredentialStatus } =
+      acceptCredentialOfferOptions
+    const client = this.getClient(agentContext)
 
-    if (credentialsToRequest?.length === 0) {
-      this.logger.warn(`Accepting 0 credential offers. Returning`)
-      return []
+    if (credentialConfigurationIds?.length === 0) {
```

Comment:
Yes because then we take all offered credentials. So we only want to catch the case here where the user provided credential configurations, but it's an empty array